internal/config/config.go

```diff
@@ -124,6 +127,7 @@ func Get() *Config {
 		options.SetDefault("db.name", "gumbaroo")
 		options.SetDefault("db.host", "0.0.0.0")
 		options.SetDefault("db.port", "5432")
+		options.SetDefault("db.impl", "mock")
```

Comment:
Not sure if it makes sense to have the mock DB be the default for local dev. It wouldn't give a realistic environment, and you would have to repopulate the db between builds. Might want to remove this.